.direnv/bin/nix-direnv-reload

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -e
+if [[ ! -d "/home/ken/Codes/ctf" ]]; then
```

Comment:
does the path need adjustments?
I have not a lot of experience with gh-actions or a like.

if it's something that should be executed locally, it'd probably make more sense to use the `$HOME` env variable